Gems/Terrain/Code/Tests/TerrainSystemBenchmarks.cpp

```diff
@@ -372,6 +384,56 @@ namespace UnitTest
         ->Args({ 4096, 1, static_cast<int>(AzFramework::Terrain::TerrainDataRequests::Sampler::EXACT) })
         ->Unit(::benchmark::kMillisecond);
 
+    BENCHMARK_DEFINE_F(TerrainSystemBenchmarkFixture, BM_ProcessHeightsRegionAsync)(benchmark::State& state)
```

Comment:
Is there a way to collapse most of this code into a single RunTerrainAsyncApiBenchmark() or something? There's a _lot_ of duplicate code across all the benchmarks below.

Reply:
Possibly, was just copying what was there for the non async versions.